howso/utilities/installation_verification.py

```diff
@@ -974,16 +994,15 @@ def _attempt_train_date_feature(result_queue: multiprocessing.Queue):
         A queue to put the results.
     """
     client = HowsoClient()
-    features = {'date': {'type': 'continuous', 'date_time_format': '%Y-%m-%d'}}
-    trainee_obj = Trainee(
-        f"installation_verification check_tzdata_installed ({get_nonce()})",
+    features = {'date': {'type': 'continous', 'date_time_format': '%Y-%m-%d'}}
```

Comment:
Erroneous change of feature type
```suggestion
    features = {'date': {'type': 'continuous', 'date_time_format': '%Y-%m-%d'}}
```

Reply:
Good catch. Addressed by: c5db5d2